install/helm/space-agon/templates/dedicated.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: "agones.dev/v1"
+kind: Fleet
+metadata:
+  name: {{ .Values.dedicated.name }}
```

Comment:
I think it would be better to use `_helpers.tpl` for common value(e.g. `space-agon`, `frontend`, `dedicated`, `director`, `mmf`, such names).

Agones helm chart's _helpers.tpl is good to know.
You can define the common template in _helpers.tpl.
https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/_helpers.tpl#L23
https://helm.sh/docs/chart_template_guide/named_templates/

For example
```
{{- define "space-agon.name" -}}
{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```

```
{{- define "dedicated.name" -}}
{{- default .Values.dedicated.name -}}
{{- end -}}
```